src/api/amazon.js

```diff
@@ -0,0 +1,18 @@
+import { getAxtionsInstace } from './api';
+import { TOPOLOGICAL_INVETORY_API_BASE } from '../constants/api-constants';
+
+// level 1
+
+export const getVms = (id) => getAxtionsInstace().get(`${TOPOLOGICAL_INVETORY_API_BASE}/sources/${id}/vms`);
+
+// level 2 vms / id / type
```

Comment:
So this is actually not entirely true. The hierarchy can be something like
```
source -> networks -> regions -> subnets -> vms -> interfaces -> tags ...
```

Now we will keep it like this for now, but we should definitely discuss how the actual topology looks like and should look like and potentially make it configurable. I recall that there is about 7 levels for the AWS source. I was testing it on was source last time.

Reply:
Yeah, I just followed the structure mentioned in the document.